common-model/src/main/java/com/sequenceiq/common/api/telemetry/model/Monitoring.java

```diff
@@ -0,0 +1,19 @@
+package com.sequenceiq.common.api.telemetry.model;
+
+import java.io.Serializable;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Monitoring implements Serializable {
```

Comment:
why do we have `Monitoring` and `MonitoringBase` with same field?

Reply:
one of it is the json that is persisted in the DB, other one is for the API, these are not always strongly the same and can change/shift from each other, now at the moment the available configs are minimal, but will be extended later